github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -423,39 +423,6 @@ postsubmits:
                 memory: "4Gi"
               limits:
                 memory: "4Gi"
-    - name: post-project-infra-prow-control-plane-deployment
-      always_run: false
-      run_if_changed: "github/ci/prow-deploy/.*"
```

Comment:
Disabling would mean removing this line, so do we really want to remove the postsubmit entirely now?

Reply:
If we want to delete the postsubmit, then we should change the PR description.

Reply:
I've updated the PR title and commit